.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,48 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `docs` label][docs-issues] (including
+        closed issues) before you continue.
+
+
+        [docs-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Adocs+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [improvement, mistake, other]
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Include as much detail as possible, for example:
+        - A (perma)link to the documentation in question
+        - Suggestions for how to change the docs
```

Minor, but for consistency this should be either

```suggestion
        - A (perma)link to the docs in question
        - Suggestions for how to change the docs
```

or


```suggestion
        - A (perma)link to the documentation in question
        - Suggestions for how to change the documentation
```